docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -1314,6 +1314,62 @@ See the proofs for this approach below.
 
 .. image:: baseline_calibration_proofs.png
 
+Summary of Iron Intervention Algorithm
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block::
+
+  # Definitions
+  t         := current time in years since start of simulation
+  t_end     := time when simulation ends, in years since start (e.g. 6 years)
+  20_weeks  := 20 weeks, measurd in years (e.g. 20*7/365.25)
+
+  # Population level parameters
+  hb_shift              = effect size on hemoglobin (normally disributed, mean 3.0 g/L)
+  bw_response           = dose-response on birthweight (normally distributed, mean 15.1/10 g/mg)
+  baseline_coverage(t)  = iron fortification coverage at time t in baseline scenario
+  delta_coverage(t)     = change in coverage from baseline to intervention
+  coverage(t)           = iron fortification coverage at time t
+                        = baseline_coverage(t) + delta_coverage(t)
+    # Multiplier on hb_shift due to age
+  hb_age_fraction(age)  = 0 if age<0.5 else (age-0.5)/1.5 if age<2 else 1
+    # Multiplier on hb_shift due to lag in response time (=1 if simulant hasn't yet received fortification)
+  hb_lag_fraction(time_since_coverage)
+                        = time_since_coverage/0.5 if 0<=time_since_coverage<0.5 else 1
+  iron_concentration    = concentration of iron in the country's fortified flour
+
+  # Individual level attributes
+  p_i = propensity of simulant and mother for exposure to unfortified flour
+  daily_flour_i = average daily flour consumption of simulant's mother
+  gestational_age_i = gestational age of simulant drawn from GBD (in years)
+  birthweight_gbd_i = birthweight drawn from GBD for simulant
+  age_i(t) = the simulant's age at time t
+  hb_gbd(age_i(t)) = hemoglobin level drawn from GBD for simulant
+  birthweight_i := simulant's birthweight after adjusting for mother's iron consumption
+  hb_i(t) := simulant's hemoglobin level after adjusting for fortification
+
+  # Stratification on child coverage - assign categories on simulant initialization
+  child_fortified_baseline_end_i = p_i < baseline_coverage(t_end)
+  child_fortified_end_i = p_i < coverage(t_end)
+
+  # At beginning of simulation
+  mother_fortified_i = "unknown"
+  mother_fortified_baseline_i = "unknown" # For stratification
+  birth_weight_i = birthweight_gbd_i
+  child_fortified_i(0) = p_i < coverage(0)
+  hb_i(0) = hb_gbd(age_i(0)) + (-coverage(0)+child_fortified_i(0)) * hb_shift *  hb_age_fraction(age_i(0))
+
+  # When simulant is born in the simulation
+  mother_fortified_i = p_i < coverage(t - gestational_age_i + 20_weeks)
+  mother_fortified_baseline_i = p_i < baseline_coverage(t - gestational_age_i + 20_weeks) # For stratification
+  bw_shift_i = bw_response * iron_concentration * daily_flour_i
+  birth_weight_i = birthweight_gbd_i + (-coverage(t)+mother_fortified_i) * bw_shift_i
```

Comment:
birth_weight_i = birthweight_gbd_i + (-coverage(t)+mother_fortified_i) * bw_shift_i

This looks appropriate to me

Reply:
Actually, it turns out there were two problems with this:
1. The equation should use `baseline_coverage(t)` instead of `coverage(t)`. This is corrected in the latest commit.
2. James says the downward shift to the treatment-deleted birthweight should be a population-level parameter rather than an individual-level parameter - i.e. it should be 
`birth_weight_i = birthweight_gbd_i - coverage(t) * average_bw_shift + mother_fortified_i * bw_shift_i`,
where `average_bw_shift` is computed by taking the mean over the population distribution of `daily_flour_i`. I'm not actually sure why this matters, but I added a Todo to change this.